### PR TITLE
8292 update emailer for Co-op IA

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#8292

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-COMPLETED.html [hunk 1]
```diff
@@ -17,17 +17,25 @@
...
+              {{ 'your Cooperative Association' if business.legalType == 'CP' else 'a business' }} 
+              with BC Business Registry.
+            </p>
 
-            [[whitespace-16px.html]]
             [[business-info.html]]
 
-            [[whitespace-24px.html]]
-            <p>Please find the following documents attached to this email:</p>
+            <p>The following documents attached to this email:</p>
```
Comment (argush3) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-COMPLETED.html:28]: missing an `are` I believe.  i.e. `<p>The following documents attached to this email:</p>` should be `<p>The following documents are attached to this email:</p>`

File: queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-PAID.html [hunk 2]
```diff
@@ -19,11 +19,9 @@
           <div class="container">
             <p>We have received your incorporation application.</p>
 
-            [[whitespace-16px.html]]
             [[business-info.html]]
 
-            [[whitespace-24px.html]]
-            <p>Please find the following documents attached to this email:</p>
+            <p>The following documents attached to this email:</p>
```
Comment (argush3) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-PAID.html:24]: missing an `are`

File: queue_services/entity-emailer/src/entity_emailer/email_templates/common/cra-notice.html [hunk 3]
```diff
@@ -1,7 +1,14 @@
 <p class="cra-notice">
-  As part of a provincial–federal partnership, BC Registries and Online Services
-  and Canada Revenue Agency (CRA) have developed an agreement to assign a Business
-  Number to all companies operating in BC. As a result of this incorporation, a
-  Business number will be assigned to the company and will be emailed to the
-  company’s registered email address.
+  {% if business.legalType == 'CP' %}
+    Now that you have completed your incorproation, please
```
Comment (argush3) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/common/cra-notice.html:3]: typo on `incorproation`.  should be `incorporation`

File: queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py [hunk 4]
```diff
@@ -164,6 +167,49 @@ def _get_pdfs(
...
+                    capture_message(f'Email Queue: filing id={filing.id}, error=certifiedRules generation',
+                                    level='error')
+                else:
+                    certified_rules_encoded = base64.b64encode(rules.content)
+                    pdfs.append(
+                        {
+                            'fileName': 'Certified Rules.pdf',
+                            'fileBytes': certified_rules_encoded.decode('utf-8'),
+                            'fileUrl': '',
+                            'attachOrder': '2'
```
Comment (argush3) [inline queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py:188]: Not entirely sure how this works but just confirming that `attachOrder` being `2` for both rules and memorandum pdf is ok.

**Comment by argush3** [comment]
looks good.  just few small things and the one question.

**Review by argush3 — APPROVED** [review]

File: queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py (hunk 4 above)
Comment (vysakh-menon-aot) [reply queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py]: Not sure either but i fixed it.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**